lib/ShareProvider.js

```diff
@@ -344,7 +350,7 @@ ShareProvider.prototype = {
       }
     };
 
-    CustomizableUI.createWidget(shareButton);
+    widget = CustomizableUI.createWidget(shareButton);
```

Comment:
Is this `widget` already declared/defined at somewhere before? I couldn't find it.